network/src/commonJvmAndroid/kotlin/com/wire/kalium/network/HttpEngine.kt

```diff
@@ -125,6 +126,11 @@ private fun OkHttpClient.Builder.ignoreAllSSLErrors() {
 private fun supportedConnectionSpecs(): List<ConnectionSpec> {
     val wireSpec = ConnectionSpec.Builder(ConnectionSpec.MODERN_TLS)
         .tlsVersions(TlsVersion.TLS_1_2)
+        .cipherSuites(
+            TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+            TLS_AES_128_GCM_SHA256,
+            TLS_AES_256_GCM_SHA384
+        )
```

Comment:
Question: the same okHttp is reused not only to connect to wire server but also to other servers (fetch custom e.g. server config), do we want to limit this change to only okhttp instances that are used to connect to wire servers?